.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "main", "master" ]
+  schedule:
+    - cron:  '*/15 * * * *'
+  workflow_dispatch:
```

Comment:
Comment: Oh nice! I just read up what this does. Nifty! On the other hand, by simply self-hosting I get this for free :P